### ISSUE TITLE
dev:source-theme:deploy symlinked theme not working 

### ISSUE THREAD

**Issue by arvenom** [magento/magento2#3313]
The app/design/frontend/Vendor/Theme folder is symlinked outside of the magento install. 
When running bin/magento dev:source-theme:deploy or bin/magento setup:static-content:deploy it will not use the symlinked files. 
If I copy it over, it works.

**Comment by DarkMukke** [comment]
i have a similar problem, we install themes though composer but for developing those themes its easier if you can just change the local files and test them out without any crazy push pulls,

so we have 2 repos, magento one and the theme one :

```
/var/www/projects/mage2/
/var/www/projects/theme-client1
```

the theme is in the composer.json required from our toran proxy, but for local development we symlink the folders, eg :

```
$ ls -l /var/www/projects/mage2/vendor/ourcompany/
total 0
lrwxrwxrwx 1 user user 47 May 16 15:22 theme-client1 -> /var/www/projects/theme-client1
```

that means any changes we make in the mage 2 app will change in the theme repo too and when ready, we just push them
when running `php bin/magento setup:static-content:deploy` we get errors like 

```
Cannot read contents from file "/var/www/projects/mage2/var/www/projects/theme-client1/Magento_Contact/templates/form.phtml"
```

I tried loading the symlink both absolute and relative but neither made a difference.

Like the OP said, when copying the files over, the static-deploy works just fine

**Comment by anton-huz** [comment]
Hi @arvenom 
We will do some investigation under MAGETWO-55445 internal ticket.

**Comment by magento-engcom-team** [comment]
@arvenom, thank you for your report.
The issue is already fixed in develop branch, 2.1.10, 2.2.1
